src/test/mochitest/head.js

```diff
@@ -452,16 +452,11 @@ async function waitForMappedScopes(dbg) {
 }
 
 function isTopFrameSelected(dbg, state) {
-  const pause = dbg.selectors.getPause(state);
-
-  // Make sure we have the paused state.
-  if (!pause) {
-    return false;
-  }
```

Comment:
Just to understand, we no longer need this because `isTopFrameSelected` is called when its already paused right?

Reply:
yeah